bundles/org.openhab.binding.ecovacs/README.md

```diff
@@ -173,3 +173,20 @@ Bridge ecovacs:ecovacsapi:ecovacsapi [ email="your.email@provider.com", password
 }
 ```
 
+## Adding support for unsupported models
+
+When encountering an unsupported model during discovery, the binding creates a log message like this one:
+
+```
+2023-04-21 12:02:39.607 [INFO ] [acs.internal.api.impl.EcovacsApiImpl] - Found unsupported device DEEBOT N8 PRO CARE (class s1f8g7, company eco-ng), ignoring.
+```
+
+In such a case, please [create an issue on GitHub](https://github.com/openhab/openhab-addons/issues), listing the contents of the log line.
+In addition to that, If the model is similar to an already supported one, you can try to add the support yourself (until getting an updated binding).
+For doing so, you can follow the following steps:
+
+- in the `<OPENHAB_USERDATA>/evocacs` folder, create a file named `custom_device_descs.json`
```

Comment:
I assume this directory is not created automatically by the binding, so the user must create it?

Reply:
Yes. I've added it as first step, I assume that was what you had in mind?